style.css

```diff
@@ -1,82 +1,317 @@
-@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins&family=Roboto:wght@700&display=swap');
 
-* {
-  padding: 0;
-  margin: 0;
+:root {
+  --dark-grey: #3c3a39;
+  --silver: #acb7c3;
+  --black: #1c1a19;
+  --orange: #ff6b00;
+  --whity: #fff5e1;
+  --blue-like: #091e42;
+  --grey: #8993a4;
+  --grey-like: rgba(255, 255, 255, 0.24);
+  --white-like: #f4f5f7;
+}
+
+*,
+*::after,
+*::before {
   box-sizing: border-box;
+  color: white;
+  text-decoration: none;
 }
 
 body {
-  min-width: 375px;
+  width: 100%;
+  font-family: "Inter", sans-serif;
 }
 
-header {
+.tool-bar {
+  width: 100%;
   display: flex;
   justify-content: space-between;
-  color: #fff;
-  background-color: #3c3a39;
-  padding: 3% 5% 3% 5%;
+  background-color: var(--dark-grey);
+  position: absolute;
+  padding: 12px 24px 12px 24px;
 }
 
-#logo {
-  font-family: "Roboto", sans-serif;
+.logo {
+  font-family: 'Roboto', sans-serif;
   font-style: normal;
-  font-weight: bold;
+  font-weight: 700;
   font-size: 18px;
   line-height: 24px;
-  text-decoration: none;
-  color: #fff;
 }
 
-#headline {
+.menu-icon {
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  color: #fff;
-  padding: 120px 5% 130px 5%;
+  padding: 120px 24px;
   gap: 12px;
-  background-image: url("./svg/side-pic.svg");
-  background-repeat: no-repeat;
-  background-attachment: fixed;
-  background-position: right bottom;
-  background-color: #1c1a19;
+  background-color: var(--black);
 }
 
-#salut {
-  color: #ff6b00;
-  font-family: "Crete Round", sans-serif;
+.hello {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
   font-weight: 400;
   font-size: 56px;
   line-height: 64px;
   letter-spacing: 0.37px;
+  color: var(--orange);
 }
 
-#job {
-  font-family: "Roboto", sans-serif;
+.iam {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
   font-weight: 700;
   font-size: 24px;
   line-height: 32px;
-  color: #fff5e1;
+  color: var(--whity);
 }
 
-#description {
-  font-family: "Poppins", sans-serif;
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
-  color: #fff;
 }
 
-#links {
+.social {
   display: flex;
   list-style-type: none;
 }
 
-.link {
-  padding: 5px;
-  background-color: transparent;
+.link:hover {
+  background: #a04300;
 }
 
-.link:hover {
-  background-color: #3c3a39;
+.background {
+  background-image: url("default/SVG/Illustration.svg");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.work {
+  display: flex;
+  align-items: center;
+  padding: 120px 24px;
+  flex-direction: column;
+  gap: 24px;
+}
```

Comment:
According to the project requirements:

In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. 😄